diff --git a/dist/chunk-W7EQS7E7.mjs b/dist/chunk-W7EQS7E7.mjs
index b74890c62705e717c63100ac02dedd4a3508126b..395011620d59c632d3406aeafe15365c07ce14aa 100644
--- a/dist/chunk-W7EQS7E7.mjs
+++ b/dist/chunk-W7EQS7E7.mjs
@@ -6,7 +6,6 @@ import { transformProps } from "@unpic/core";
 var nestedKeys = /* @__PURE__ */ new Set(["style"]);
 var fixedMap = {
   srcset: "srcSet",
-  fetchpriority: "fetchPriority"
 };
 var camelize = (key) => {
   if (key.startsWith("data-") || key.startsWith("aria-")) {
diff --git a/dist/index.js b/dist/index.js
index 7c0eebba1352f63d6fee2d20ac7c00813f2d646c..716cda22031249fa35eb8bdc21fc58a7f15b5faa 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -43,7 +43,6 @@ var import_core = require("@unpic/core");
 var nestedKeys = /* @__PURE__ */ new Set(["style"]);
 var fixedMap = {
   srcset: "srcSet",
-  fetchpriority: "fetchPriority"
 };
 var camelize = (key) => {
   if (key.startsWith("data-") || key.startsWith("aria-")) {
